### ISSUE TITLE
New icon for pool cleaning robot

### ISSUE THREAD

**Issue by Nordicfastware** [Templarian/MaterialDesign#6393]
### Before submitting a new icon request, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://materialdesignicons.com/) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](http://dev.materialdesignicons.com/roadmap/brand-icons) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I only intend to request a single icon (or a few near-identical ones) in this issue.

### What would you use this icon for?

Pool cleaning robot, to go along with vaccum and lawn mower robots.

### Give us an example of what this icon might look like.

https://www.vectorstock.com/royalty-free-vector/robotic-pool-cleaner-stylized-icon-automatic-vector-30917253

or

https://www.dreamstime.com/smart-pool-cleaner-icon-set-flat-line-style-smart-pool-cleaner-icon-set-flat-line-style-modern-cleaning-underwater-image173975798

**Comment by Gamester17** [comment]
Yes please add icons for robotic pool cleaner robot (similar to match the existing lawn mover robot but with tracks instead of wheels).

I would use it in Home Assistant as per this request for a new pool_cleaner entity domain and/or device class as a new building block integration for platforms:

* https://github.com/orgs/home-assistant/discussions/3662

If possible add icons for "Pool Cleaner Robot" + "Pool Cleaner Robot On" + "Pool Cleaner Robot Offline" respectivly,

Here is some inspiration for Pool Cleaner Robot

<img width="192" height="128" alt="Image" src="https://github.com/user-attachments/assets/b9664bfd-f77d-4499-a695-6f4ca810d1d9" />

<img width="256" height="256" alt="Image" src="https://github.com/user-attachments/assets/76829ad1-43e1-4fe2-99e4-988670e4a0de" />

For reference, Adobe also have a very nice stock icon for a Pool Cleaner Robot but that is not royalty free

* https://stock.adobe.com/images/robotic-pool-cleaner-stylized-icon-automatic-cleaning-robot-for-pool-service-symbol-adjustable-stroke-width/343495059

<img width="645" height="403" alt="Image" src="https://github.com/user-attachments/assets/bb40c5e4-ea03-4f63-a4e3-6f1c2fac96cf" />

Here are also other pool cleaner icons from shutterstock for reference:

* https://www.shutterstock.com/sv/image-vector/pool-service-line-icons-set-linear-2471231091

Here is also a picture of some real pool cleaner robots for referece:

<img width="827" height="423" alt="Image" src="https://github.com/user-attachments/assets/5793d34a-adad-42b9-a644-529eb9295fed" />
